dev/PushNotifications/GetNotificationEventArgs.h

```diff
@@ -25,11 +26,14 @@ namespace winrt::Microsoft::Windows::PushNotifications
                 return winrt::make<winrt::Microsoft::Windows::PushNotifications::implementation::PushNotificationReceivedEventArgs>(payloadAsWstring);
             }
         }
+        return GetArgsFromComStore();
+    }
+}
 
-        const DWORD receiveArgsTimeoutInMSec{ 2000 };
-        THROW_HR_IF(HRESULT_FROM_WIN32(ERROR_TIMEOUT), !GetWaitHandleForArgs().wait(receiveArgsTimeoutInMSec));
-
-        // If COM static store was uninit, let it throw
-        return winrt::Windows::ApplicationModel::Core::CoreApplication::Properties().Lookup(ACTIVATED_EVENT_ARGS_KEY);
+namespace winrt::Microsoft::Windows::ToastNotifications
+{
+    static winrt::Windows::Foundation::IInspectable Deserialize(winrt::Windows::Foundation::Uri const& /* uri */)
```

Comment:
Why can't we reuse the method above instead of defining this again?

Reply:
I wanted to have a deserialize function for ToastNotifications in case we needed some special parsing, but after looking at it again we can reuse the method.